src/Microsoft.ApplicationInsights/Microsoft.ApplicationInsights.csproj

```diff
@@ -28,8 +28,8 @@
     <!--Copyright defined above-->
     <!--Copyright></Copyright-->
     <RequireLicenseAcceptance>true</RequireLicenseAcceptance>
-    <PackageLicenseUrl>http://go.microsoft.com/fwlink/?LinkID=510709</PackageLicenseUrl>
-    <PackageProjectUrl>http://go.microsoft.com/fwlink/?LinkId=392727</PackageProjectUrl>
+    <PackageLicenseUrl>https://go.microsoft.com/fwlink/?LinkID=510709</PackageLicenseUrl>
```

Comment:
I wonder should `owner` property refer to AppInsightsSDK as well now? Or Microsoft is sufficient?

Reply:
Author MUST be Microsoft only.
For Owner, it is recommended but not required that we use Microsoft and also the Nuget owner, AppInsightsSDK.

@SergeyKanzhelev Since this wasn't required, I was saving it for later but if you agree, I can add it now. :)